msp430-gcc.rb

```diff
@@ -30,7 +30,8 @@ def install
     # which, upon further inspection, arises when xgcc bails out when it sees
     # this argument.
     ENV.remove_from_cflags '-Qunused-arguments'
-    ENV.remove_from_cflags '-march=native'
```

Comment:
Yes, this flag should also be removed, the regex `/ ?-march=\S*/` should also match the `-march=native` flag.